core-plugins/src/main/java/io/cdap/plugin/batch/joiner/JoinerConfig.java

```diff
@@ -104,6 +114,22 @@
     "and this value is false, records with a null id from input stages A and B will not get joined together.")
   private Boolean joinNullKeys;
 
+  @Nullable
+  @Macro
+  @Description(DISTRIBUTION_FACTOR_DESC)
+  private Integer distributionFactor;
+
+  @Nullable
+  @Macro
+  @Description(DISTRIBUTION_STAGE_DESC)
+  private String distributionStageName;
+
+  @Macro
+  @Nullable
+  @Description("Distribution is useful when the input data is skewed, enabling distribution will allow you to "
```

Comment:
skewed. Enabling will increase the parallelism

Reply:
done